### PR TITLE
Resolves: #62 - Correctly serializes list and dictionary in query

### PR DESCRIPTION
**Pull request by jterry75**
Signed-off-by: Justin Terry <juterry@microsoft.com>

### CODE REVIEW

**Comment by jterry75** [comment]
@jstarks @swernli - Looking for feedback here but this handles the issues that we are seeing for enumerables and maps.

File: Docker.DotNet/EnumerableQueryStringConverter.cs [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+            return typeof (IEnumerable).IsAssignableFrom(t);
+        }
+
+        public string[] Convert(object o)
+        {
+            Debug.Assert(o != null);
+            Debug.Assert(o is IEnumerable);
+
+            var items = new List<string>();
+            foreach (var e in ((IEnumerable) o))
```
Comment (jstarks) [inline Docker.DotNet/EnumerableQueryStringConverter.cs]: Seems like this could be a lot simpler with linq

Comment (jterry75) [inline Docker.DotNet/EnumerableQueryStringConverter.cs]: Unfortunately LINQ only works on IEnumerable<T> and I dont see why we need a generic restriction on EnumerableQueryStringConverter but If you want I can make it EnumerableQueryStringConverter<T> and then on the attributes generate typeof(EnumerableQueryStringConverter<T>) that matches.

Comment (jstarks) [inline Docker.DotNet/EnumerableQueryStringConverter.cs]: ehh not worth it

**Comment by jstarks** [comment]
LGTM

**Comment by jterry75** [comment]
Verified that this now properly handles maps/lists.

**Pull request merged**